trackpy/linking.py

```diff
@@ -546,6 +548,14 @@ def link_df(features, search_range, memory=0,
     if hash_size is None:
         MARGIN = 1  # avoid OutOfHashException
         hash_size = features[pos_columns].max() + MARGIN
+    # Check if DataFrame is writeable.
+    # I don't know how to do this for pandas < 0.16.
+    if (StrictVersion(pd.__version__) >= StrictVersion('0.16.0') and
+        features.is_copy is not None and not copy_features):
```

Comment:
sorry, picky white space complaint, can you indent this a bit more?  I think the sin of messing up the continuation indentation is far less than the sin of aligning with the next indentation block.